projects/components/src/filtering/filter-button/filter-button.component.ts

```diff
@@ -62,10 +64,13 @@ export class FilterButtonComponent implements OnChanges {
   ) {}
 
   public ngOnChanges(): void {
-    this.availableFilters =
-      !isNil(this.attribute) && !isNil(this.value)
-        ? this.buildAvailableFilters(this.attribute, this.value, this.subpath)
-        : [];
+    if (isNil(this.attribute)) {
+      this.availableFilters = [];
+    } else if (isNil(this.value)) {
+      this.availableFilters = this.buildSimplifiedFilters(this.attribute);
```

Comment:
if value is undefined or null, then we want to show only eq and not_eq filters?

Reply:
In this case, is receiving undefined as a value. I think that is the same if we receive null or undefined. Instead of showing the blank popover with this change, we show equals/not equals to null.

Reply:
@aaron-steinfeld requested this change. Do you think that it's fine. I'm not finding a case where this logic doesn't work well.

Reply:
Two filters added after to discuss with Aaron offline

Reply:
Basically equality filters (=, !=, in, not_in) always make sense for any value, null included. Type specific filters (numeric comparisons, regex, contains etc.) should not be shown for nulls.